Add hosting team faq (#409)
WordPress/hosting-handbook: get-involved.md

## First Steps for New Contributors

> New to the Hosting Team or joining a Contributor Day? Start with the [Hosting Team FAQ](https://make.wordpress.org/hosting/handbook/get-involved/faq/) for quick answers to common onboarding questions, including Git and GitHub basics.

Reviewer: ```suggestion
New to the Hosting Team or joining a Contributor Day? New contributors can [start here](https://make.wordpress.org/hosting/handbook/get-involved/#first-steps-for-new-contributors, and be sure to check the [Hosting Team FAQ](https://make.wordpress.org/hosting/handbook/get-involved/faq/) for quick answers to common onboarding questions, including Git and GitHub basics.
```
